packages/column-views/src/unit-details/panel.ts

```diff
@@ -456,14 +489,20 @@ function IntervalProportions({ unit }) {
     p0 = h("span.joint-proportion", [p0, h("span.sep", "to"), p1]);
   }
 
+  const clickable = onClick != null;
+
   return h("div.interval-proportions", [
     h(IntervalTag, {
+      className: "int_id-" + interval0?.int_id + (clickable ? " clickable" : ""),
```

Comment:
Not sure why you have to add stuff to the `className`? It seems to have to do with how you're trying to infer what element to link to, but this is pretty awkward.